respect-lib-shared/src/commonMain/kotlin/world/respect/shared/domain/report/model/ReportSeries.kt

```diff
@@ -0,0 +1,96 @@
+package world.respect.shared.domain.report.model
+
+import kotlinx.datetime.DatePeriod
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+import org.jetbrains.compose.resources.StringResource
+import world.respect.shared.generated.resources.Res
+import world.respect.shared.generated.resources.*
+
+@Serializable
+data class ReportSeries(
+
+    val reportSeriesUid: Int = 0,
+
+    val reportSeriesTitle: String = "Series 1",
+
+    @Contextual
+    val reportSeriesYAxis: Indicator = DefaultIndicators.list.first(),
+
+    val reportSeriesVisualType: ReportSeriesVisualType = ReportSeriesVisualType.BAR_CHART,
+
+    val reportSeriesSubGroup: ReportXAxis? = ReportXAxis.DAY,
+
+    val reportSeriesFilters: List<ReportFilter>? = null,
+
+    )
+
+@Serializable(with = IndicatorSerializer::class)
+data class Indicator(
+    val name: String? = null,
```

Comment:
Clearly name and sql at least should not be nullable Description also should not be nullable.